src/main/java/duke/functionality/Ui.java

```diff
@@ -0,0 +1,57 @@
+package duke.functionality;
+
+import java.util.Scanner;
+
+import duke.exception.BlankCommandException;
+
+/**
+ * Represents the User Interface of the Duke project. A <code> Ui </code> object corresponds
+ * to all user interface displays. Such as the messages.
+ */
+public class Ui {
+    private static final String GREETING = "Hello! I'm TaskJamie\nWhat can i do for you?";
+    private static final String ENDING = "Bye. Hope to see you again soon!";
```

Comment:
Exit String was already returned in the ExitCommand. Is there a need for it here as well?